src/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react"
+import { ApiArticles } from "../Api"
+
+function Articles() {
+
+    const [articles, setArticles] = useState();
+
+    useEffect(() => {
+        ApiArticles().then((pendingArticles) => {
+            setArticles(pendingArticles.articles)
+        })
+    }, []);
+    if (!articles) {
+        return <p>loading...</p>
+    };
+
+    return (
+        <>
```

Comment:
A nested div inside a fragment is not the best way to present this component. A section would be a more semantic tag.